docs/docs-content/clusters/edge/cluster-management/agent-upgrade-airgap.md

```diff
@@ -0,0 +1,111 @@
+---
+sidebar_label: "Upgrade Palette Agent on Airgap Clusters"
+title: "Upgrade Palette Agent on Airgap Clusters"
+description: "Learn how to upgrade the Palette agent on airgap clusters. "
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge", "architecture"]
+---
+
+In connected clusters, the Palette agent gets upgraded automatically with the Palette version, unless you explicitly
+[pause upgrades](../../cluster-management/platform-settings/pause-platform-upgrades.md). In airgap clusters, this does
+not happen automatically. When you want to upgrade the agent version, you can include the new agent version in a new
+cluster profile, and upgrade the cluster using the new profile.
+
+## Prerequisites
+
+- An active Edge cluster deployed in airgap mode.
+
+- Linux Machine (Physical or VM) with an AMD64 architecture.
+
+- Palette API key. Refer to the [User Authentication](../../../user-management/authentication/api-key/create-api-key.md)
+  resource to learn how to create a Palette API key.
+
+- An Edge Native type cluster profile. Refer to
```

Comment:
```suggestion
- An Edge type cluster profile. Refer to
```
Product told us to not use the term. Edge Native in our docs. We really need to get on the same page as a company because UX is not adhering to this 😞